### ISSUE TITLE
WCAG2Mobile: Draft feedback - 2.4.6 Headings and Labels

### ISSUE THREAD

**Issue by ChrisLoiselle** [w3c/wcag2ict#669]
Mary Jo:

- No comment for now, but will need to watch progress on issue #50. 

Chris L: 

- The discussion as of April 2 2025 was to possibly introduce notes on labels and headings. I personally don't think it is necessary to define these unless they deviate from web. The intent of the SC is that headings and labels describe topic or purpose. 
- https://www.w3.org/TR/wcag2mobile-22/#success-criterion-2-4-6-headings-and-labels
- https://github.com/w3c/matf/issues/50

**Comment by pday1** [comment]
Closed - insufficient time or resource to review each WCAG2Mobile clause
